base/post-receive.sample

```diff
@@ -0,0 +1,43 @@
+#!/bin/sh
+#
+# An example hook script to update the working tree, including its
+# submodules, after receiving a push.
+#
+# This hook requires core.worktree to be explicitly set, and
+# receive.denyCurrentBranch to be set to false.
+#   git config --path core.worktree ../
+#   git config --bool receive.denyCurrentBranch false
+#
+# To enable this hook, rename this file to "post-receive"
+# and run `chmod +x post-receive`.
+
+# Read standard input or hook will fail
+while read oldrev newrev refname
+do
+:
+done
+
+# Unset GIT_DIR or the universe will implode
+unset GIT_DIR
+
+# Set appropriate configuration to allow this to work
+git config --path core.worktree ../
+git config --bool receive.denyCurrentBranch false
+
+# Change directory to the working tree; exit on failure
+cd `git config --get core.worktree` || exit
+
+# Force checkout
+git checkout --force
+
+# Force update submodules
+git submodule update --init --recursive --force
+
+# Install the latest version of circulation-web
+if [ -d api/admin ]; then
+  cd api/admin
+  npm install
```

Comment:
This isn't good for production, it will install all the npm dependencies as well as the one built js file we actually need.

Is it common to use git for deployment this way? It works well for python since you can deploy the source code as is, but I guess I was thinking we'd build a docker container with the python code and bundled js already in it.